service/resource/configmap/delete.go

```diff
@@ -57,7 +57,7 @@ func (r *Resource) NewDeletePatch(ctx context.Context, obj, currentState, desire
 	return patch, nil
 }
 
-func (r *Resource) newDeleteChange(ctx context.Context, obj, currentState, desiredState interface{}) (interface{}, error) {
+func (r *Resource) newDeleteChangeForDeletePatch(ctx context.Context, obj, currentState, desiredState interface{}) (interface{}, error) {
```

Comment:
There is a delete change relevant for delete events and a delete change relevant for update events. The method naming reflects that. The difference is very small within the filter logic of the implementation but it is necessary.